Inventory_Management_System/Utility/Logger.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Inventory_Management_System.Utility;
+    public static class Logger
```

Comment:
u can define an `ILogger` interface with a Log method, and then implement this interface in a `ConsoleLogger` class. 
by doing so, the service calling the Log method will be decoupled from the specific implementation details of how and where the logging is performed. 
this approach follows the Dependency Inversion Principle (DIP)